backends-velox/src/main/scala/org/apache/gluten/backendsapi/velox/VeloxBackend.scala

```diff
@@ -222,8 +223,10 @@ object VeloxBackendSettings extends BackendSettingsApi {
 
     def validateFileFormat(): Option[String] = {
       format match {
-        case _: ParquetFileFormat => None
-        case _: FileFormat => Some("Only parquet fileformat is supported in Velox backend.")
+        case _: ParquetFileFormat => None // Parquet is directly supported
+        case h: HiveFileFormat if h.toString.contains("parquet") => None // Parquet via Hive SerDe
+        case _ =>
+          Some("Only parquet fileformat is supported in Velox backend.") // Unsupported format
```

Comment:
Can we add one more config to control if we transofrm hive file format ? We can enable it by defualt.